### PR TITLE
[PROJQUAY-753] Reduce logging noise

### PR DESCRIPTION
**Pull request by kurtismullins**
### Description of Changes

Reduce signal to noise ratio in Quay's default logging configuration.

#### Changes:

* Only use `info` level logging for information that is useful for system/application administrators and is not internal, implementation-level details.
* Change the build manager to emit `info` level logging instead of `debug` by default.
* Do not use `print()` statements for logging purposes.

#### Issue:

- https://issues.redhat.com/browse/PROJQUAY-753

**TESTING**

- Ensure enough useful information is logged.
- Ensure the build manager's logging is not emitting any `debug` log messages.

**BREAKING CHANGE**

None.

---

## Reviewer Checklist

- [ ] It works!
- [ ] Comments provide sufficient explanations for the next contributor
- [ ] Tests cover changes and corner cases
- [ ] Follows Quay syntax patterns and format

### CODE REVIEW

File: conf/init/data_migration.py [hunk 1]
```diff
@@ -18,7 +22,7 @@ def current_migration():
 
 
 def main():
-    print(current_migration())
+    logger.info(current_migration())
```
Comment (josephschorr) [inline conf/init/data_migration.py:25]: This will break container startup; this output is piped into the alembic command

File: data/database.py [hunk 2]
```diff
@@ -393,14 +393,14 @@ def _db_from_url(
         driver = drivers.pooled_driver
         db_kwargs["stale_timeout"] = db_kwargs.get("stale_timeout", None)
         db_kwargs["max_connections"] = db_kwargs.get("max_connections", None)
-        logger.info(
+        logger.debug(
```
Comment (josephschorr) [inline data/database.py:396]: Leave these as info

File: data/logs_model/__init__.py [hunk 3]
```diff
@@ -26,9 +26,7 @@ def __init__(self):
 
     def initialize(self, model):
         self._model = model
-        logger.info("===============================")
-        logger.info("Using logs model `%s`", self._model)
-        logger.info("===============================")
+        logger.debug("Using logs model `%s`", self._model)
```
Comment (josephschorr) [inline data/logs_model/__init__.py:29]: Leave this and the next one as info

File: data/registry_model/__init__.py [hunk 4]
```diff
@@ -15,6 +15,4 @@ def __getattr__(self, attr):
 
 
 registry_model = RegistryModelProxy()
-logger.info("===============================")
-logger.info("Using registry model `%s`", registry_model._model)
-logger.info("===============================")
+logger.debug("Using registry model `%s`", registry_model._model)
```
Comment (josephschorr) [inline data/registry_model/__init__.py:18]: Leave as-is

File: data/secscan_model/__init__.py [hunk 5]
```diff
@@ -30,10 +30,8 @@ def configure(self, app, instance_keys, storage):
 
         self._v4_namespace_whitelist = app.config.get("SECURITY_SCANNER_V4_NAMESPACE_WHITELIST", [])
 
-        logger.info("===============================")
-        logger.info("Using split secscan model: `%s`", [self._model, self._v4_model])
-        logger.info("v4 whitelist `%s`", self._v4_namespace_whitelist)
-        logger.info("===============================")
+        logger.debug("Using split secscan model: `%s`", [self._model, self._v4_model])
```
Comment (josephschorr) [inline data/secscan_model/__init__.py:33]: Leave as-is. These data model startup logs are only written once per process

File: external_libraries.py [hunk 6]
```diff
@@ -141,7 +141,7 @@ def _download_url(url):
                 filename = os.path.basename(url).split("?")[0]
 
             path = os.path.join(local_directory, filename)
-            print("Downloading %s to %s" % (url, path))
+            logger.info("Downloading %s to %s" % (url, path))
```
Comment (josephschorr) [inline external_libraries.py:144]: Leave this as a print; it is invoked by the build

File: tools/sendresetemail.py [hunk 7]
```diff
@@ -1,7 +1,7 @@
-from app import app
+import logging
```
Comment (josephschorr) [inline tools/sendresetemail.py:1]: Anything under `tools` is intended to be run from the command line; please leave these as `print`s

File: util/backfillreplication.py [hunk 8]
```diff
@@ -14,6 +14,9 @@
 from util.registry.replication import queue_storage_replication
 
 
+logger = logging.getLogger(__name__)
```
Comment (josephschorr) [inline util/backfillreplication.py:17]: This is a CLI tool so please leave as `print`s

File: util/migrate/cleanup_old_robots.py [hunk 9]
```diff
@@ -32,15 +32,15 @@ def cleanup_old_robots(page_size=50, force=False):
             namespace, _ = parsed
             if namespace in encountered_namespaces:
                 if not encountered_namespaces[namespace]:
-                    logger.info("Marking %s to be deleted", robot.username)
+                    logger.info("Marking robot %s to be deleted", robot.username)
```
Comment (josephschorr) [inline util/migrate/cleanup_old_robots.py:35]: Ditto

File: util/verifybackfill.py [hunk 10]
```diff
@@ -19,7 +19,7 @@ def _vs(first, second):
 
 
 def verify_backfill(namespace_name):
-    logger.info("Checking namespace %s", namespace_name)
+    logger.debug("Checking namespace %s", namespace_name)
```
Comment (josephschorr) [inline util/verifybackfill.py:22]: This file should have been deleted in my cleanup PR.... Please just delete it or revert this change and I'll delete it in a followup

File: util/verifyplacements.py [hunk 11]
```diff
@@ -46,7 +46,7 @@ def verify_placements():
 
         encountered.add(storage_row.id)
 
-        logger.info("Checking placements for storage `%s`", storage_row.uuid)
+        logger.debug("Checking placements for storage `%s`", storage_row.uuid)
```
Comment (josephschorr) [inline util/verifyplacements.py:49]: This is a CLI tool; leave as-is

File: data/logs_model/__init__.py (hunk 3 above)
Comment (kurtismullins) [reply data/logs_model/__init__.py:29]: @josephschorr Do you want to include those log lines which only include `=` characters? If so, want me to just make one line string and include line breaks?

**Pull request merged**